scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/internal/semanticdb/DocumentOps.scala

```diff
@@ -253,6 +253,12 @@ trait DocumentOps { self: DatabaseOps =>
               val mname = mctorrefs(gpoint)
               gtree match {
                 case g.Select(_, g.nme.CONSTRUCTOR) => success(mname, gtree.symbol)
+                case gtree: g.ClassDef if config.members.isAll =>
```

Comment:
should we additionally guard against cases when it's not an anonymous class? My suggested test was an attempt to try and break this

Reply:
Yep you are right:

```scala
members(
    """
      |object a {
      |  trait Foo { def bar: Int }  
      |  class Bar() extends Foo { def bar = 1 }  
      |  val a = new Bar()
      |}""".stripMargin,
    ""
  )
```

```
[info]   ===========
[info]   => Obtained
[info]   ===========
[info]   _empty_.a.Foo#{
[info]     bar.
[info]   }
[info]   
[info]   =======
[info]   => Diff
[info]   =======
[info]   -_empty_.a.Foo#{
[info]   -bar.
[info]   -}
```